.github/workflows/zulip-tests.yml

```diff
@@ -27,11 +27,15 @@ jobs:
           tools/lint --skip=gitlint
 
   test:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest, windows-latest]
         python-version: [3.6, 3.7, 3.8, 3.9]
+        exclude:
+          - os: windows-latest
+            python-version: 3.6 # cryptography install fails on Windows with python 3.6 since pip is quite old.
```

Comment:
The error was something along "upgrade pip to resolve the error". Specifically, it occurred only on 3.6 on Windows. So maybe old pip is the reason for it. There are other workarounds to make it running on 3.6, but since support for it is going to drop soon, I skipped them.

Reply:
Yeah, I think probably the added benefit of specifically testing 3.6 on Windows is small enough that it's probably not worth fixing this in the ~6 months of 3.6 support still left.